examples/build.sh

```diff
@@ -0,0 +1,2 @@
+# to be run in scout/examples
+bjam toolset=clang dhtd=off cxxflags="-std=c++14"
```

Comment:
This file should be called something like "build.macos.sh" to make it clear that this is platform specific.